### ISSUE TITLE
Picker not announcing placeholder text when items selected

### ISSUE THREAD

**Issue by alexwaibel** [microsoft/fluentui#16099]
### Environment Information

- **Package version(s)**: @fluentui/react 7.129.0
- **Browser and OS versions**: Windows 10, Edge Version 86.0.622.69

### Describe the issue:

When there are elements selected and the picker combo box becomes focused, only the selected item list is read. If the input has placeholder text, this text is not read to the user. Items can be preselected, so it's possible for a user to land on this input for the first time and not know that they may use arrow keys to select the text input box to enter more items and tab to the next input, accidentally skipping the picker.

### Please provide a reproduction of the issue in a codepen:

Here's a [codepen repro](https://codepen.io/alwaibel/pen/jOMPwqE) using the ["People Picker with Preselected Items" example from the docs](https://fabricweb.z5.web.core.windows.net/oufr/7.153.2/#/examples/peoplepicker). The only change I made was to add a placeholder string to the input box.

#### Actual behavior:

On picker focus when there are selected items and a placeholder string, only the selected items are announced.

#### Expected behavior:

On picker focus when there are selected items and a placeholder string, selected items **and** the placeholder string are announced.

**Comment by JustSlone** [comment]
@alexwaibel thanks for filing this! 

This looks related to #15182 probably same root issue between the two pickers. 

Looking into this, it seems that this is only repro'ing in Narrator. NVDA seems to handle this as expected reading the place holder text when the edit box is navigated to. Is this the behavior expected here?
![people-placeholder](https://user-images.githubusercontent.com/40304143/100965059-b771ba80-34de-11eb-9e3b-404c980c6cf7.gif)

**Comment by alexwaibel** [comment]
@JustSlone not sure if I'm missing something, but using "Tab" while running  NVDA does not allow me to move through the list items and to the input. It simply skips straight to the checkboxes. Here's the behavior I'm seeing:

![Screen Recording 2020-12-04 at 4 25 56 PM](https://user-images.githubusercontent.com/7026879/101227898-6254b680-364e-11eb-9d5a-42c2fbbd7750.gif)

Same behavior going backwards using "Shift + Tab". Goes straight to the first list item.

**Comment by JustSlone** [comment]
Yes, that is expected. Tab brings you to the list, then the left/right arrow keys move through it. Sorry for not mentioning that. 

This should help:
![picker-keyboard](https://user-images.githubusercontent.com/40304143/101268461-ee7ee080-3717-11eb-86ba-42e5a4358c36.gif)
